lib/Doctrine/Adapter/Statement/Oracle.php

```diff
@@ -583,7 +583,7 @@ private function parseQuery($query=null)
         }
         $bind_index = 1;
         // Replace ? bind-placeholders with :oci_b_var_ variables
-        $query = preg_replace("/(\?)/e", '":oci_b_var_". $bind_index++' , $query);
+        $query = preg_replace_callback("/(\?)/", function($m) { return ":oci_b_var_". $bind_index++; } , $query);
```

Comment:
Argh. Do you have a fix / possibility to test one?
Feel free to submit a PR to fix that (and if you put some test, it could be awesome)